### DISCUSSION TITLE
"No Recommended" not working for new account

### DISCUSSION THREAD

**Discussion by avespanthera** [AprilSylph/XKit-Rewritten#366] [Help Desk]
i hope this is the right place to post this... apologies if it's not.

i just made a new tumblr account and in spite of having "No Recommended" on, turning off dashboard settings ("Best Stuff First," "Include stuff in your orbit" and "Include followed tag posts" are disabled) and following a couple blogs, i still see recommended posts/blogs on my dashboard--but only when logged in to the new account. i'm even seeing posts i've told tumblr i'm not interested in when i refresh or go back to the dashboard from another page.

the sidebar and radar are properly hidden, and i don't think i'm seeing recommended tags or anything else, just recommended posts. i also have Tumblr Savior installed with the options to hide such things checked, but i haven't seen any conflicts on the other account yet.

i'm following 5 blogs on my old account, 2 on the new. i made a test post on the new account, no change; the old one has 242 posts.

have i missed something? why is this only happening with the new account? google hasn't given me any good answers.

any help is appreciated!

**Comment by AprilSylph** [comment]
[No Recommended](https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#no-recommended) doesn't block recommended posts. I don't know what undismissable recommendation type you're encountering, so I suppose I would have to create a third Tumblr account myself to find out.

> i'm even seeing posts i've told tumblr i'm not interested in when i refresh

It seems odd that Tumblr has an undismissable recommendation type at all, so this could actually be a Tumblr bug. If you plan to follow more blogs, it would be very helpful to know if they stop appearing when you do. If you don't, I would recommend you [contact Tumblr support](https://www.tumblr.com/support) to get a clear answer on
1. if you should be able to dismiss this recommendation type, and
2. the conditions for it to stop appearing.

**Reply by avespanthera** [reply]
wait, if it doesn't hide the recommendations i'm seeing, then what does the "Hide recommended blogs between posts" do? is that just confusion over terminology here? i'm seeing posts from blogs i don't follow on my dashboard with green "Recommended" text next to the tags, which i can manually dismiss one by one that reappear if i refresh unless i block them.

if these can't be blocked by XKit or Tumblr Savior, do you know of a way to hide them?

**Reply by AprilSylph** [reply]
"Hide recommended blogs between posts" hides recommended blog carousels between posts.

![image](https://user-images.githubusercontent.com/28949509/145929409-8c49b425-9874-4e1c-9947-ddf9b88157ad.png)

> do you know of a way to hide them?

Nope! This type of recommended post is so rare that I was convinced they didn't exist. I could conceivably work on an option to hide them (assuming creating a new Tumblr account gives me the same recommendations) but I'm guessing they're only appearing because you follow so few blogs, which is a bit of an edge case. Again, I would love if you could contact Tumblr support to confirm this.

**Reply by avespanthera** [reply]
hm, well i blocked all the posts i was seeing so those are gone and have yet to be replaced. but if i see more i guess i'll contact support.

**Comment by AprilSylph** [comment]
No Recommended now includes the on-by-default option "Hide recommended posts on the dashboard" starting with XKit Rewritten v0.13.1, which targets all recommended post types other than pinned posts and suggestions from followed tags.
